dll/win32/shell32/wine/shell32_main.c

```diff
@@ -1158,7 +1199,19 @@ static INT_PTR CALLBACK AboutDlgProc( HWND hWnd, UINT msg, WPARAM wParam, LPARAM
                 WCHAR szAuthorsText[20];
 
                 // Preload the ROS bitmap
-                hLogoBmp = (HBITMAP)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDB_REACTOS), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
+                //hLogoBmp = (HBITMAP)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDB_REACTOS), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
```

Comment:
This doesn't make any sense. Whilst the whole file's code comes from Wine, without a doubt, this specific line is definitely not part of Wine (Wine doesn't even show the ROS logo originally, which I hope this is self-explanatory). 

Thoroughly searching within Git history using `git blame`, the code was added by this commit -- [`6ced137`](https://github.com/reactos/reactos/commit/6ced137ce2f6f415c2106ce8b57d936002cf5941). Just removing this line should be enough as it was added by us, not Wine.

Reply:
From [WINESYNC.txt](https://github.com/reactos/reactos/blob/master/media/doc/WINESYNC.txt)
```
dll/win32/shell32             # Forked at Wine-20071011
```
So there is no much point of using `#ifdef __REACTOS__` in shell32.

Reply:
I didn't know this was a forked file. Thanks for pointing it out.